test/integration/crud/abstract_operation.test.ts

```diff
@@ -353,7 +353,7 @@ describe('abstract operation', async function () {
               subclassType.name === 'ProfilingLevelOperation' ? { ok: 1, was: 1 } : { ok: 1 };
             const cmdCallerStub = sinon
               .stub(Server.prototype, 'command')
-              .yieldsRight(undefined, yieldDoc);
+              .returns(Promise.resolve(yieldDoc));
```

Comment:
applies to all occurrences in this PR:

```suggestion
              .resolves(yieldDoc);
```

Reply:
done!